tests/analyzer/basic.zeek

```diff
@@ -1,9 +1,53 @@
 # Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
 
-# @TEST-EXEC: zeek -r ${TRACES}/dhcp.pcap %INPUT
+# @TEST-EXEC: zeek -r ${TRACES}/dhcp.pcap %INPUT > basic.out
+# @TEST-EXEC: btest-diff basic.out
 # @TEST-EXEC: btest-diff conn.log
 # @TEST-EXEC: btest-diff dhcp.log
 #
 # @TEST-DOC: Test DHCP analyzer with small trace.
 
+
 @load analyzer
+
+# Trigger: DHCP Discover message
+event dhcp_discover(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP discover";
+    }
+
+# Trigger: DHCP Offer message
+event dhcp_offer(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP offer";
+    }
+
+# Trigger: DHCP Request message
+event dhcp_request(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP request";
+    }
+
+# Trigger: DHCP Decline message
+event dhcp_decline(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP decline";
+    }
+
+# Trigger: DHCP Ack message
+event dhcp_ack(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP ack";
+    }
+
+# Trigger: DHCP Nak message
+event dhcp_nak(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
+    {
+    print "DHCP nak";
+    }
+
+# Trigger: DHCP Release message
+event dhcp_release(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options)
```

Comment:
Like I already wrote for the EVT file, let's get rid of these events since they are also not part of the built-in DHCP analyzer. Let's instead implement a verbose `dhcp_message` handler here so we can test what we parsed,

```suggestion

event dhcp_message(c: connection, is_orig: bool, msg: DHCP::Msg, options: DHCP::Options) {
    print is_orig, msg, options;
}
```

Reply:
Removed

Reply:
Let's implement the test printing the produced `msg` from a `dhcp_message` handler.

Reply:
In progress

Reply:
Done, I implemented the `print is_orig, msg, options;` as you suggested before. Had to change btest baseline accordingly.